### PR TITLE
Adding ActionConfig options

### PR DESCRIPTION
**Pull request by m2bright**

### CODE REVIEW

File: rapid/lib/work_request.py [hunk 1]
```diff
@@ -56,19 +57,23 @@ def prepare_to_send(self, data_in):
...
-                    tmp = attr
-                    if tmp == '_grain':
-                        tmp = 'grain'
-                    setattr(self, attr, data_in[tmp])
+                    if attr == '_grain':
+                        for g_attr in ['_grain', 'grain']:
+                            if g_attr in data_in:
+                                setattr(self, '_grain', data_in[g_attr])
+                    else:
+                        setattr(self, attr, data_in[attr])
```
Comment (BambooBryson) [inline rapid/lib/work_request.py]: Is this in both PRs @m2bright ?

File: rapid/master/data/migrations/env.py [hunk 2]
```diff
@@ -38,8 +38,9 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-target_metadata = db.Model.metadata
-
+target_metadata = get_declarative_base().metadata
+# print(target_metadata)
```
Comment (BambooBryson) [inline rapid/master/data/migrations/env.py]: you want this left in here commented out?

Comment (m2bright) [reply rapid/master/data/migrations/env.py]: Nope! 👍 Thanks.

File: rapid/lib/work_request.py (hunk 1 above)
Comment (m2bright) [reply rapid/lib/work_request.py]: Yes. I needed it is elsewhere.

File: rapid/master/data/migrations/env.py (hunk 2 above)
Comment (m2bright) [inline rapid/master/data/migrations/env.py]:
```suggestion
```

**Pull request merged**